src/pocketmine/Player.php

```diff
@@ -3590,6 +3591,25 @@ public function sendTranslation($message, array $parameters = []){
 		$this->dataPacket($pk);
 	}
 
+	public function sendWeather(){
```

Comment:
I don't like this here. It adds extra overhead for broadcasting purposes. Player also doesn't need any more mess in it.